frontend/src/components/mine/Permit/MinePermitInfo.js

```diff
@@ -1,166 +1,126 @@
-      return date1 === date2 ? order1.order_no - order2.order_no : date1 - date2;
+    this.props.openModal({
+      props: {
+        onSubmit,
+        title,
+      },
+      content: modalConfig.ADD_PERMIT,
     });
-
-const getPermitteeName = (permittees) =>
-  permittees[0] ? permittees[0].party.party_name : Strings.EMPTY_FIELD;
-
-const transformRowData = (permit, partyRelationships) => {
-  const latestAmendment = permit.amendments[0];
-  const firstAmendment = permit.amendments[permit.amendments.length - 1];
-
-  const permittees = getPermittees(partyRelationships, permit);
-  const permitteeName =
-    partyRelationships.length === 0 ? Strings.LOADING : getPermitteeName(permittees);
-
-  return {
-    key: permit.permit_guid,
-    lastAmended: (latestAmendment && formatDate(latestAmendment.issue_date)) || Strings.EMPTY_FIELD,
-    permitNo: permit.permit_no || Strings.EMPTY_FIELD,
-    firstIssued: (firstAmendment && formatDate(firstAmendment.issue_date)) || Strings.EMPTY_FIELD,
-    permittee: permitteeName,
-    authorizationEndDate:
-      (latestAmendment && formatDate(latestAmendment.authorization_end_date)) ||
-      Strings.EMPTY_FIELD,
-    amendments: permit.amendments,
-    status: permit.permit_status_code,
   };
-};
-
-const transformChildRowData = (amendment, index, amendments) => ({
-  amendmentNumber: amendments.length - index,
-  receivedDate:
-    (amendment.received_date && formatDate(amendment.received_date)) || Strings.EMPTY_FIELD,
-  issueDate: (amendment.issue_date && formatDate(amendment.issue_date)) || Strings.EMPTY_FIELD,
-  authorizationEndDate:
-    (amendment.authorization_end_date && formatDate(amendment.authorization_end_date)) ||
-    Strings.EMPTY_FIELD,
-  description: Strings.EMPTY_FIELD,
-});
 
-export const MinePermitInfo = (props) => {
-  const amendmentHistory = (record) => {
-    const childRowData = record.amendments.map(transformChildRowData);
-    return (
-      <Table align="left" pagination={false} columns={childColumns} dataSource={childRowData} />
-    );
+  handleAddPermit = (values) => {
+    const payload = { mine_guid: this.props.mine.guid, ...values };
+    payload.permit_no = `${values.permit_type}${values.permit_activity_type || ""}-${
+      values.permit_no
+    }`;
+    this.props.closeModal();
+    return this.props.createPermit(payload).then(() => {
+      this.props.fetchPermits({ mine_guid: this.props.mine.guid });
+    });
   };
-  const rowData = props.mine.mine_permit.map((permit) =>
-    transformRowData(permit, props.partyRelationships)
-  );
 
-  return (
-    <Table
-      className="nested-table"
-      align="left"
-      pagination={false}
-      columns={columns}
-      dataSource={rowData}
-      expandedRowRender={amendmentHistory}
-      locale={{ emptyText: <NullScreen type="permit" /> }}
-    />
-  );
-};
+  render() {
+    return [
+      <div>
+        <div className="inline-flex between">
+          <div />
```

Comment:
Empty div?

Reply:
It expands to fill empty space in the flex-box.